drivers/spi/spi_ll_stm32.c

```diff
@@ -1186,6 +1186,14 @@ static int transceive_dma(const struct device *dev,
 	/* This is turned off in spi_stm32_complete(). */
 	spi_stm32_cs_control(dev, true);
 
+	uint8_t word_size_bytes = bits2bytes(
+			SPI_WORD_SIZE_GET(config->operation));
```

Comment:
Nitpicking: fits in a single line:
```suggestion
	uint8_t word_size_bytes = bits2bytes(SPI_WORD_SIZE_GET(config->operation));